dashboard/assets/js/components/product.js

```diff
@@ -343,3 +343,20 @@ export function PrintModeToggle({isPrintMode, onClick}) {
     </a>
   );
 }
+
+/**
+ * Export product component
+ */
+export class ExportProduct extends Component {
```

Comment:
this class can be just a function, as it only has a `render` method like this https://github.com/ministryofjustice/moj-product-dashboard/blob/214f78ca1c2af6c8b0caad205e1f203360589b37/dashboard/assets/js/components/product.js#L296